source/MarianaTrench.cpp

```diff
@@ -423,8 +423,9 @@ std::vector<std::string> filter_existing_jars(
 
 void MarianaTrench::run(const program_options::variables_map& variables) {
   Context context;
-
-  context.options = std::make_unique<Options>(variables);
+  std::string json_file_path = variables["config"].as<std::string>();
+  std::unique_ptr<Options> options_from_file = Options::options_from_json_file(json_file_path);
+  context.options = std::move(options_from_file);
```

Comment:
you can avoid the temporary variable:
```suggestion
  context.options = Options::options_from_json_file(json_file_path);
```